### PR TITLE
error500

### PR DESCRIPTION
**Pull request by Kelvin-Ofili**
error500(UI)
error404(under maintenance)(UI)

### CODE REVIEW

File: src/assets/vectors/navBars.svg
Comment (The-Anomaly) [inline src/assets/vectors/navBars.svg:1]: Why is this asset here? What was it used for?

File: src/layouts/home/navbar/index.tsx
Comment (The-Anomaly) [inline src/layouts/home/navbar/index.tsx:1]: Going to need you to fix the indentation of your editor. 
Do you use prettier?

File: src/router/routeBuilder.ts
Comment (The-Anomaly) [inline src/router/routeBuilder.ts:1]: No direct changes to this page, but your indentation has affected it.
Please use prettier.

File: src/router/routes.ts
Comment (The-Anomaly) [inline src/router/routes.ts:1]: Indentation problem. Use prettier.

File: yarn.lock
Comment (The-Anomaly) [inline yarn.lock:1]: How come there are changes to the lock file?

**Review by The-Anomaly — CHANGES_REQUESTED** [review]

**Comment by Kelvin-Ofili** [comment]
Hi Favour,

My bad. It was for the nav-bar of the Unsupported screen size page, but you
told me not to bother about the nav-bar. I forgot to remove it.

Yes, I use prettier.
To trigger the prettier action, I just need to press ctrl s, no? Will do
that.

For the yarn.lock issue, I have literally no idea why there are changes in
it. All I remember using Yarn to do was to start up the app at some point.
Maybe that may have caused the change. I will look into it.

I should be done with these within the hour.
Thank you.

**Review by The-Anomaly — DISMISSED** [review]
Although the indentation issue persists, I'll be approving this PR.
I'll set up a prettier configuration for the project, so there's uniformity.

**Pull request merged**